assignments/assignment7/faulty-oops.md

```diff
@@ -0,0 +1,44 @@
+Unable to handle kernel NULL pointer dereference at virtual address 0000000000000000
```

Comment:
@anjo6767 use markdown formatting for this file - see how it renders today at https://github.com/cu-ecen-aeld/assignments-3-and-later-anjo6767/blob/17665d00bfb00ecff2d97909f55a3a45f9b6ee29/assignments/assignment7/faulty-oops.md